.github/workflows/publish.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         uses: actions/setup-java@5ffc13f4174014e2d4d4572b3d74c3fa61aeb2c2 # v3.11.0
         with:
           distribution: 'corretto'
-          java-version: 8
+          java-version: 11
```

Comment:
You're changing this so that when we eventually merge `v2` into main, we change the publish JVM, right? Said otherwise - we don't want to merge this change into `main` too until then (unlike `pr_build.yaml`).

Reply:
If this is the case, do we also need to change `pr_build` etc as well, so that when we merge v2 into `main` , it ends up setup correctly?